gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -102,6 +106,8 @@ MM_VerboseHandlerOutput::enableVerbose()
 {
 	/* Initialized */
 	(*_mmOmrHooks)->J9HookRegisterWithCallSite(_mmOmrHooks, J9HOOK_MM_OMR_INITIALIZED, verboseHandlerInitialized, OMR_GET_CALLSITE(), (void *)this);
+	(*_mmOmrHooks)->J9HookRegisterWithCallSite(
+			_mmOmrHooks, J9HOOK_MM_OMR_REINITIALIZED, verboseHandlerReinitialized, OMR_GET_CALLSITE(), (void *)this);
```

Comment:
ref: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#old-code
there is existing code which does not follow the rule to spread across multiple lines.
consistency with old code will supersede the rule to spread across multiple lines.
for this PR, my recommendation will be not to spread them across multiple lines.